release/models/acl/openconfig-packet-match.yang

```diff
@@ -313,6 +319,21 @@ module openconfig-packet-match {
         "Source IPv4 address prefix.";
     }
 
+    leaf masked-source-address {
+      type oc-inet:ipv4-address;
+      description
+        "Source IPv4 address, to be used with source-wildcard-mask.
+        This configuration is used in cases of matches that cannot 
+        be expressed by a CIDR mask..";
+    }
```

Comment:
Here and throughout I suggest more explicit language:

```suggestion
        "Source IPv4 address, to be used with source-wildcard-mask.
        This node must only be used if a masked address is required.
        If this node is populated, ../source-address must not be used.";
    }
```

Reply:
Done